CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ For upgrade instructions, please check the [migration guide](MIGRATIONS.md).
 [Commits](https://github.com/scalableminds/webknossos/compare/19.12.0...HEAD)
 
 ### Added
--
+- Added new viewing permission for annotations: public, internal, private
```

Comment:
Could elaborate a bit to explain what each option entails? This should also be in the documentation (screenshot can be updated together with your other PR, but the text can be updated in this PR).